### DISCUSSION TITLE
Custom fields

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/kimai#4955] [Plugins]
We want to (regularly) import out tasks from redmine to track work times on them. The task plugin seems an overkill to me, because we don't want to make a real task planning in kimai. We're happy with redmine for this. So, the users should not make or take tasks or anything.
I think about a custom field of type `choice-list` with the selectable redmine task id to log times for. So far this works fine, but I didn't find an API point to update the custom entries of the field. Is there one?

**Comment by pkoevesdi** [comment]
Ah, also I'm missing the option to assign a task to a time entry.

**Comment by kevinpapst** [comment]
<img width="805" alt="Bildschirmfoto 2024-07-06 um 19 23 54" src="https://github.com/kimai/kimai/assets/533162/67d4389e-4607-4b07-9abb-5c3462db80f8">

**Comment by pkoevesdi** [comment]
That I know of, but that's not what I mean. I want to update the entries of the field, that the user can select from. So, that there is always the current list of tickets.

**Comment by pkoevesdi** [comment]
Also, I thought of putting my redmine projects and tasks into "Projects" and "Activities". But these fields are mandatory, which shouldn't be in our company, see https://github.com/kimai/kimai/discussions/4721#discussioncomment-9975568
